ReleaseNotes.md

```diff
@@ -3,7 +3,7 @@ Release Notes
 
 OpenRocket 22.00.beta.01
```

Comment:
Maybe already change this to 22.02.beta.01

Reply:
Party pooper.

Reply:
Is that what the build number is going to be?  All the screenshots are for 22.00.beta.01, and I'll kill myself before re-doing all those.